core/cont/src/TList.cxx

```diff
@@ -714,7 +714,7 @@ void TList::RecursiveRemove(TObject *obj)
    while (lnk) {
       next = lnk->Next();
       TObject *ob = lnk->GetObject();
-      if (ob->TestBit(kNotDeleted)) {
+      if (ob && ob->TestBit(kNotDeleted)) {
```

Comment:
I can format it later to whatever is the consensus, but what really matters is deciding if we should apply this patch or not. Also because I found more places where the same problem exists.